app/controllers/events/view/tickets/orders/list.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  columns: [
+    {
+      propertyName   : 'user',
+      template       : 'components/ui-table/cell/events/view/tickets/orders/cell-order',
+      title          : 'Order',
+      disableSorting : true
+    },
+    {
+      propertyName : 'amount',
+      template     : 'components/ui-table/cell/events/view/tickets/orders/cell-amount',
+      title        : 'Total Amount'
+    },
+    {
+      propertyName : 'tickets.length',
+      title        : 'Quantity'
+    },
+    {
+      propertyName   : 'user',
```

Comment:
@CosmicCoder96 We can directly pass the email here `user.email`, we do not need an extra template for it.